src/internal/packager/sbom/viewer/viewer.js

```diff
@@ -1,36 +1,36 @@
-const sbomSelector = document.getElementById('sbom-selector')
-const distroInfo = document.getElementById('distro-info')
-const modal = document.getElementById('modal')
-const modalFader = document.getElementById('modal-fader')
-const modalTitle = document.getElementById('modal-title')
-const modalContent = document.getElementById('modal-content')
-const artifactsTable = document.createElement('table')
+const sbomSelector = document.getElementById("sbom-selector")
+const distroInfo = document.getElementById("distro-info")
+const modal = document.getElementById("modal")
+const modalFader = document.getElementById("modal-fader")
+const modalTitle = document.getElementById("modal-title")
+const modalContent = document.getElementById("modal-content")
+const artifactsTable = document.createElement("table")
 
-document.body.appendChild(artifactsTable)
+document.body.appendChild(artifactsTable);
 
 function initSelector() {
     window.location.href
```

Comment:
Know this isn't code you changed, but what the heck did we/I leave this here before?

Reply:
I deleted it when I copied it but forgot to go back and clean it up.  Looked like a miscopy to me